operations/patch_util.go

```diff
@@ -289,6 +293,53 @@ func (p *patchParams) loadProject(conf *ClientSettings) error {
 	return nil
 }
 
+func (p *patchParams) setLocalAliases(conf *ClientSettings) error {
+	if p.Alias != "" {
+		for i := range conf.Projects {
+			if conf.Projects[i].Name == p.Project {
+				if errStrs := model.ValidateProjectAliases(conf.Projects[i].LocalAliases, "Local Aliases"); len(errStrs) != 0 {
+					return errors.Errorf("validating local aliases: '%s'", errStrs)
+				}
+				for _, alias := range conf.Projects[i].LocalAliases {
+					if alias.Alias == p.Alias {
+						p.addAliasToPatchParams(alias)
+					} else {
+						return errors.Errorf("both variants and tasks not defined in the alias '%s'", alias.Alias)
+					}
+				}
+			}
+		}
+	}
+	return nil
+}
+
+// addAliasToPatchParams add the matching local alias to patch params.
+func (p *patchParams) addAliasToPatchParams(alias model.ProjectAlias) {
+	if (alias.Variant != "" || alias.VariantTags != nil) && (alias.Task != "" || alias.TaskTags != nil) {
```

Comment:
We don't need this overall if statement now that we validate I think.

Reply:
yeah we dont